tests/fixtures/kdc.py

```diff
@@ -0,0 +1,71 @@
+import itertools
+import pytest
+import sdk_auth
+import sdk_hosts
+
+from tests.fixtures.hdfs import HDFS_SERVICE_NAME, GENERIC_HDFS_USER_PRINCIPAL, ALICE_PRINCIPAL
+from tests.fixtures.kafka import KAFKA_SERVICE_NAME
+
+
+@pytest.fixture(scope='session')
+def kerberos_env():
+    try:
+        kafka_principals = build_kafka_principals()
+        hdfs_principals = build_hdfs_principals()
+
+        kerberos_env = sdk_auth.KerberosEnvironment(persist=True)
+        kerberos_env.add_principals(kafka_principals + hdfs_principals)
+        kerberos_env.finalize()
+        yield kerberos_env
+
+    finally:
+        print('noop')
```

Comment:
```suggestion
```

Reply:
this wasn't intended to be committed, removed